dbms/src/Flash/Pipeline/PipelineEvent.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
As I understand the `PipelineEvent`, it is actually to represent the **plain** pipeline execution, which is the default pipeline model we understand. But I notice that you are willing to implement some other event to represent the **fine-grained** pipeline model, which has a different DOP and specified partition number and different dependency. Though the `Pipeline` structure is strong enough to execute both, the naming needs to differentiate them. So I suggest renaming it to `PlainPipelineEvent` and use `FineGrainedPipelineEvent` for the future one.

Reply:
good point, updated.